packages/diagrams/lib/src/material_app.dart

```diff
@@ -46,6 +47,14 @@ class MaterialAppDiagram extends StatelessWidget implements DiagramMetadata {
           debugShowCheckedModeBanner: false,
         );
         break;
+      case _textstyle:
+        returnWidget = const MaterialApp(
+          title: 'Material App',
```

Comment:
not needed

Reply:
I meant that the MaterialApp's title wasn't needed, not the entire switch clause.  It's necessary to provide code to generate the image per https://github.com/flutter/assets-for-api-docs/#generation